splash/qtrender_lua.py

```diff
@@ -1094,9 +1169,12 @@ def get_wrapped(self):
         return self._wrapped
 
     def run_async_command(self, cmd):
-        """ Execute _AsyncBrowserCommand """
-        meth = getattr(self.tab, cmd.name)
-        return meth(**cmd.kwargs)
+        """ Execute _AsyncBrowserCommand or _AsyncCallbackCommand"""
+        if isinstance(cmd, AsyncBrowserCommand):
+            meth = getattr(self.tab, cmd.name)
+            return meth(**cmd.kwargs)
+        elif isinstance(cmd, AsyncFunctionCommand):
+            cmd.kwargs["callback"]()
```

Comment:
is it intentional that callback retval is ignored here? is it intentional that the parameter is still called "callback" here?

should we put a `raise TypeError` at the end of the function itself to catch unhandled cases?